providers/evercrypt/fst/EverCrypt.Hash.fst

```diff
@@ -134,13 +134,16 @@ let init #a s =
 
 friend Vale.SHA.SHA_helpers
 
+// Avoid a cross-compilation unit symbol visibility... duplicate locally.
```

Comment:
ah, excellent, I see the fix now -- this is great, and will allow marking the constants as static 👍

Reply:
Ok, I trust you on this :)